plugins/sql-language/src/main/java/org/polypheny/db/sql/language/fun/SqlCastFunction.java

```diff
@@ -47,7 +47,7 @@
 
 
 /**
- * SqlCastFunction. Note that the std functions are really singleton objects, because they always get fetched via the StdOperatorTable. So you can't store any local info in the class
+ * SqlCastFunction. Note that the std functions are really singleton objects, because they always get fetched via the StdOperatorTable. So you can't storeId any local info in the class
```

Comment:
This seems to be a find and replace error

Reply:
Adjusted.